From bc947d548d73ea7522f3b53eb5622f241874af6a Mon Sep 17 00:00:00 2001
From: "qiufang.dai" <qiufang.dai@amlogic.com>
Date: Wed, 28 May 2014 10:31:11 +0800
Subject: [PATCH 4231/5965] vpu: don't disable/enable vpu in supend/resume
 patch

To fix below issue:

[  551.954873@0] suspend mesonfb+ @ 1325, parent: platform
[  551.960098@0] suspend vpu+ @ 1325, parent: platform
EEEE I3000000032940xf100110003877500EEEE I400000004294_M8_BL1_10221D113B323EEEE I3000000032940xbbbb000026A22525
no sdio debug board detected
TE : 1442304
BT : 12:47:54 May 16 2014
DCDC01 set to 1050, register from 0x0018 to 0x0024, addr:0x0036
---
 arch/arm/mach-meson8/vpu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-meson8/vpu.c b/arch/arm/mach-meson8/vpu.c
index c8adf4e713cd..ab82d50de163 100755
--- a/arch/arm/mach-meson8/vpu.c
+++ b/arch/arm/mach-meson8/vpu.c
@@ -652,13 +652,13 @@ static void vpu_driver_disable(void)
 #ifdef CONFIG_PM
 static int vpu_suspend(struct platform_device *pdev, pm_message_t state)
 {
-	vpu_driver_disable();
+	//vpu_driver_disable();
 	return 0;
 }
 
 static int vpu_resume(struct platform_device *pdev)
 {
-	vpu_driver_init();
+	//vpu_driver_init();
 	return 0;
 }
 #endif
-- 
2.19.0

